ocaml/idl/datamodel.ml

```diff
@@ -5532,6 +5539,7 @@ let pif =
                                                                                                                            		can not be used, nor can the interface be bonded or have VLANs based on top through xapi." ~default_value:(Some (VBool true));
       field ~lifecycle:[Published, rel_creedence, ""] ~qualifier:DynamicRO ~ty:(Map(String, String)) ~default_value:(Some (VMap [])) "properties" "Additional configuration properties for the interface.";
       field ~lifecycle:[Published, rel_dundee, ""] ~qualifier:DynamicRO ~ty:(Set(String)) ~default_value:(Some (VSet [])) "capabilities" "Additional capabilities on the interface.";
+      field ~lifecycle:[Published, rel_inverness, ""] ~qualifier:DynamicRO ~ty:pif_igmp_status ~default_value:(Some (VEnum "na")) "igmp_snooping_enabled" "The mcast_snooping_enable status of the corresponding network bridge";
```

Comment:
We can call the field `igmp_snooping_status`.

Reply:
And for the doc string: "The IGMP snooping status..." (not "mcast_snooping_enable").